anago

```diff
@@ -200,6 +200,46 @@ check_acls () {
 }
 
 ###############################################################################
+# Ensures all registries that will be used during both mock and --nomock
+# runs allow write access so we don't fall over later
+# @param registries - A space separated list of registries
+#
+ensure_registry_acls () {
```

Comment:
Both of these registries are now writable via the same production user we use to create actual releases that publish to google_containers.